tests/contacts.test.js

```diff
@@ -18,10 +21,11 @@ beforeAll(async () => {
   await broker.loadService(path.resolve(__dirname, './services/synchronizer.service.js'));
   await broker.loadService(path.resolve(__dirname, './services/profiles.app.js'));
   await broker.loadService(path.resolve(__dirname, './services/contacts.app.js'));
+  await broker.loadService(path.resolve(__dirname, './services/events.app.js'));
 
   // Mock notification service
   await broker.createService({
-    mixins: [require('./services/notification.service')],
+    mixins: [notificationFilter, require('./services/notification.service')],
```

Comment:
Your integration tests passed because you put the mixins on the correct order here :-)